R/methods.R

```diff
@@ -157,6 +157,7 @@ tidy.SingleCellExperiment <- function(object) {
 #' @importFrom purrr map
 #' @importFrom purrr set_names
 #' @importFrom purrr list_transpose
+#' @importFrom tidybulk as_SummarizedExperiment
```

Comment:
this package has a utility function as_SummarizedExperiment

https://github.com/stemangiola/tidySingleCellExperiment/blob/d22e6ec47ddbb733228787a0e11102765092a629/R/tidybulk_utilities.R#L23

this is to not have tidybulk as dependency

Reply:
Makes sense. From what I can tell the fails are due to tidybulk being a missing dependency during the vignette build process. 
<img width="1611" alt="Screenshot 2023-11-03 at 09 45 22" src="https://github.com/stemangiola/tidySingleCellExperiment/assets/29175188/1cb81b6e-0b18-42ee-b488-a8db81b750f4">

Reply:
I have now got 3 different build fails depending on platform. Windows is the least fussy and gives me: 

    0 errors ✔ | 1 warning ✖ | 2 notes ✖
    Execution halted
    Error: Process completed with exit code 1.

On Ubuntu I get: 

    >Run BiocManager::install(version = "3.18", ask = FALSE, force = TRUE)
    Error: Bioconductor version '3.18' requires R version '4.3'; use `version = '3.19'`
      with R version 4.4; see https://bioconductor.org/install
    Execution halted
    Error: Process completed with exit code 1.

And for Mac I get a couple:

        Error: --- re-building ‘introduction.Rmd’ using rmarkdown
    The magick package is required to crop "/private/var/folders/3s/vfzpb5r51gs6y328rmlgzm7c0000gn/T/Rtmp3GMShx/Rbuild22e839638fa1/tidySingleCellExperiment/vignettes/introduction_files/figure-html/plot1-1.png" but not available.
    The magick package is required to crop "/private/var/folders/3s/vfzpb5r51gs6y328rmlgzm7c0000gn/T/Rtmp3GMShx/Rbuild22e839638fa1/tidySingleCellExperiment/vignettes/introduction_files/figure-html/plot2-1.png" but not available.
    The magick package is required to crop "/private/var/folders/3s/vfzpb5r51gs6y328rmlgzm7c0000gn/T/Rtmp3GMShx/Rbuild22e839638fa1/tidySingleCellExperiment/vignettes/introduction_files/figure-html/unnamed-chunk-12-1.png" but not available.
    
    Quitting from lines 198-210 [preprocess] (introduction.Rmd)
    Error: Error: processing vignette 'introduction.Rmd' failed with diagnostics:
    function 'as_cholmod_sparse' not provided by package 'Matrix'
    --- failed re-building ‘introduction.Rmd’
    
    SUMMARY: processing the following file failed:
      ‘introduction.Rmd’
    
    Error: Error: Vignette re-building failed.
    Execution halted
    Error: Error: Failed to install 'tidySingleCellExperiment' from local:
      ! System command 'R' failed
    Execution halted
    Error: Process completed with exit code 1.

To be honest none of these seem to be related to my PR but I could be wrong.

Reply:
That's annoying, I would say that if one system succeeds that's Ok with us.

It seems that windows has the chance to succeed please solve the warning and that should be enough.